app/assets/javascripts/discourse/app/components/bookmark-list.js

```diff
@@ -1,17 +1,53 @@
+import Component from "@ember/component";
+import { action } from "@ember/object";
+import { schedule } from "@ember/runloop";
+import bootbox from "bootbox";
+import discourseDebounce from "discourse-common/lib/debounce";
+import { openBookmarkModal } from "discourse/controllers/bookmark";
 import {
   openLinkInNewTab,
   shouldOpenInNewTab,
 } from "discourse/lib/click-track";
-import Component from "@ember/component";
+import Scrolling from "discourse/mixins/scrolling";
 import I18n from "I18n";
 import { Promise } from "rsvp";
-import { action } from "@ember/object";
-import bootbox from "bootbox";
-import { openBookmarkModal } from "discourse/controllers/bookmark";
 
-export default Component.extend({
+export default Component.extend(Scrolling, {
   classNames: ["bookmark-list-wrapper"],
 
+  didInsertElement() {
+    this._super(...arguments);
+    this.bindScrolling();
+    this.scrollToLastPosition();
+  },
+
+  willDestroyElement() {
+    this._super(...arguments);
+    this.unbindScrolling();
+  },
+
+  scrollToLastPosition() {
+    let scrollTo = this.session.bookmarkListScrollPosition;
+    if (scrollTo && scrollTo >= 0) {
+      schedule("afterRender", () => {
+        discourseDebounce(
+          this,
+          function () {
+            if (this.element && !this.isDestroying && !this.isDestroyed) {
+              $(window).scrollTop(scrollTo + 1);
```

Comment:
can we avoid jQuery? I believe all browsers now support smooth scrolling on vanilla JS's `scrollTo`.

Reply:
I do not think this was ever supposed to be a smooth scrolling. I replaced jQuery usage here.